Index: backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/DM.java
===================================================================
diff --git a/backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/DM.java b/backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/Dm.java
rename from backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/DM.java
rename to backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/Dm.java
--- a/backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/DM.java	(revision a814a9058573631085c186aef18085cfd113d66f)
+++ b/backend/devstagram/src/main/java/com/devstagram/domain/dm/entity/Dm.java	(date 1773715836252)
@@ -2,13 +2,21 @@
 
 import com.devstagram.domain.user.entity.User;
 import com.devstagram.global.entity.BaseEntity;
+import jakarta.persistence.Entity;
+import lombok.Getter;
+import lombok.Setter;
 
-public class DM extends BaseEntity {
+@Entity
+@Getter
+@Setter
+public class Dm extends BaseEntity {
 
-    ChatRoom chatRoom;
+    DmRoom dmRoom;
 
     User sender;
 
+    String type;
+
     String content;
 
     boolean isRead;
