Index: .idea/modules/devstagram.main.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module version=\"4\">\r\n  <component name=\"AdditionalModuleElements\">\r\n    <content url=\"file://$MODULE_DIR$/../../backend/devstagram/build/generated/sources/annotationProcessor/java/main\">\r\n      <sourceFolder url=\"file://$MODULE_DIR$/../../backend/devstagram/build/generated/sources/annotationProcessor/java/main\" isTestSource=\"false\" generated=\"true\" />\r\n    </content>\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules/devstagram.main.iml b/.idea/modules/devstagram.main.iml
--- a/.idea/modules/devstagram.main.iml	(revision 5633d8b13a0605d5757c9c74fc034b40c84f99aa)
+++ b/.idea/modules/devstagram.main.iml	(date 1773810475173)
@@ -5,4 +5,7 @@
       <sourceFolder url="file://$MODULE_DIR$/../../backend/devstagram/build/generated/sources/annotationProcessor/java/main" isTestSource="false" generated="true" />
     </content>
   </component>
+  <component name="CheckStyle-IDEA-Module" serialisationVersion="2">
+    <option name="activeLocationsIds" />
+  </component>
 </module>
\ No newline at end of file
